lib/ReactViews/Tools/DiffTool/DiffTool.tsx

```diff
@@ -362,6 +355,35 @@ class Main extends React.Component<MainPropsType> {
     );
   }
 
+  @action
+  async componentDidMount() {
+    // Look for any existing marker like from a search result and filter
+    // imagery at that location
+    const markerLocation = getMarkerLocation(this.props.terria);
+    const sourceItem = this.props.sourceItem;
+    if (
+      this.location === undefined &&
+      markerLocation &&
+      Mappable.is(sourceItem)
+    ) {
+      const part = sourceItem.mapItems.find(p => ImageryParts.is(p));
+      const im = part && ImageryParts.is(part) && part.imageryProvider;
```

Comment:
Just realised this now as it's called `im` in various parts of difftool.tsx but it can be confusing for an `ImageryProvider` to be abbreviated to `im`

Reply:
Yes! I did it for a cleaner perttier formatting, but it is not worth it :)